web-stories.php

```diff
@@ -40,8 +40,8 @@
 	exit;
 }
 
-define( 'WEBSTORIES_VERSION', '1.8.0-alpha.0' );
-define( 'WEBSTORIES_DB_VERSION', '3.0.10' );
+define( 'WEBSTORIES_VERSION', '1.9.0-alpha.0' );
+define( 'WEBSTORIES_DB_VERSION', '3.0.9' );
```

Comment:
This needs to be changed back to 3.0.10 ⚠️

Reply:
Fixed